src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -138,6 +140,18 @@ public Biome(ModElement element) {
 
 	}
 
+	public boolean haveFruits() {
+		String[] airBlocks = { "blocks.air", "blocks.cave_air", "blocks.void_air" };

```

Comment:
Mappings are fixed, so no need to lowercase here to compare, just use the right names from the datalists.

Also change to hasFruite as you did for vines, biome is one, so it is Has biome fuits

Reply:
blocks.air is not the name used in datalists, so this will not work